src/ReadTrainer/PlayText.js

```diff
@@ -106,12 +106,17 @@ const PlayText = inject("store", "morsePlayer")(
         if (!this.props.store.morse.playing) {
           if (this.playCount === 0) {
             this.playText();
+          } else if (this.playCount >= this.props.store.morse.maxRepeats || this.replayCount >= this.props.store.morse.maxRepeats) {
+            this.stop();
+
+          // no longer effective
           } else if (this.playCount > 20 || this.replayCount > 20) {
```

Comment:
Can you please remove this whole block given that it's no longer useful?

(bonus points for also removing all of the onAbort plumbing since it's no longer useful)